test/suite/test_txn16.py

```diff
@@ -92,11 +92,10 @@ def run_toggle(self, homedir):
                 cur_logs = fnmatch.filter(os.listdir(homedir), "*gerLog*")
                 scur = set(cur_logs)
                 sorig = set(orig_logs)
-                # There should never be overlap with the log files that
-                # were there originally.  Mostly this checks that after
-                # opening with logging disabled and then re-enabled, we
-                # don't see log file 1.
-                self.assertEqual(scur.isdisjoint(sorig), True)
+                # There can be overlap with the log files that were
```

Comment:
I didn't understand this change properly, can you add a comment to explain it?

Reply:
In this test file it deletes all the log files and then reopen `wiredtiger_open` with logging and check if no log files are created. 
But with the existing change during recovery `wiredtiger_open` RTS is called and a log file is generated with which the test was failing, I changed the test to expect a log file after `wiredtiger_open`.